frontend/webadmin/modules/gwt-common/src/main/java/org/ovirt/engine/ui/common/widget/uicommon/popup/AbstractVmPopupWidget.java

```diff
@@ -1755,7 +1756,6 @@ private void setNumaInfoMsg(String message) {
     }
 
     private void enableNumaFields(boolean enabled) {
-        numaNodeCount.setEnabled(enabled);
         numaSupportButton.setEnabled(enabled);
```

Comment:
What about the NUMA Pinning dialog button? Is it still required for the VM to be pinned to show it?

Reply:
Yes, I think this is what `numaSupportButton` is about.
We have two fields, one for the panel, one for the vNUMA count.

Reply:
I guess we still need the host to be pinned to show the host topology in the dialog.